modules/library/main/src/main/java/org/geotools/filter/function/FilterFunction_strURLEncode.java

```diff
@@ -0,0 +1,62 @@
+package org.geotools.filter.function;
```

Comment:
header required for new file.

The "FilterFunction_strURLEncode was just a naming convention used by a code generator, please consider renaming the class to follow java contentions - ``URLEncodeFunction``.